.hooks/post-checkout

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "updating daml submodule"
+git submodule update daml
```

Comment:
we should also document the initial checkout of the submodule in the readme

Reply:
should actually work via initDev!